xml/zypper.xml

```diff
@@ -935,13 +935,13 @@ systemctl status sshd</screen>
     removerepo</command> together with the alias or number of the repository
     you want to delete. For example, to remove the repository
     <literal os="sles;sled">SLEHA-12-GEO</literal><literal
-    os="osuse">Leap-42.3-NOSS</literal>
+    os="osuse">Leap-15.3-NOSS</literal>
```

Comment:
That repo is called `repo-non-oss` or `Non-OSS Repository` respectively, on a freshly installed 15.3 machine anyway. No version number and definitely not "NOSS".